pretalx_lower_thirds/signals.py

```diff
@@ -1,8 +1,19 @@
 from django.dispatch import receiver
 from django.urls import resolve, reverse
-from django.utils.translation import ugettext_lazy as _
+from django.utils.translation import gettext_noop, ugettext_lazy as _
+from i18nfield.strings import LazyI18nString
+from pretalx.common.models.settings import hierarkey
 from pretalx.orga.signals import nav_event_settings
 
+hierarkey.add_default(
+    "lower_thirds_no_talk_info",
+    LazyI18nString.from_gettext(
+        gettext_noop("Sorry, there's currently no talk running")
```

Comment:
Does this also fix #2?

Reply:
Sure does! The default value will be returned by `hierarkey` for every implicit null value.